package.json

```diff
@@ -8,7 +8,7 @@
   },
   "dependencies": {
     "del": "^5.1.0",
-    "digitalmarketplace-frameworks": "https://github.com/alphagov/digitalmarketplace-frameworks.git#v17.9.0",
+    "digitalmarketplace-frameworks": "git+https://github.com/alphagov/digitalmarketplace-frameworks.git#v17.10.0",
```

Comment:
I don't think the `git+` is needed here?

```suggestion
    "digitalmarketplace-frameworks": "https://github.com/alphagov/digitalmarketplace-frameworks.git#v17.10.0",
```

Reply:
@lfdebrux think you're probably right but this is what got generated by running: 
```
npm install https://github.com/alphagov/digitalmarketplace-frameworks.git#v17.10.0 --save
``` 
so I assume this is OK, rather than doing the change by hand.

Reply:
👍